src/main/resources/db/changelog/changes/create-transaction-table.xml

```diff
@@ -87,4 +87,17 @@
     </createTable>
   </changeSet>
 
+  <changeSet id="addPickupLocationData" author="MagzhanArtykov">
```

Comment:
as we're still in the dev stage, do we really need to bring such changes via a brand-new change set, or let's just adjust the existing createTable descriptor?

@gurleenkaurbp, @MagzhanArtykov 
Suggest pls

Reply:
Yes its in dev stage no need to create a change set as of now. But if modules start to get released every sprint then change set can be used.

Reply:
Removed change set